.github/workflows/run-cctp-tests.yml

```diff
@@ -50,8 +37,10 @@ jobs:
   build:
     name: "Build"
     runs-on: ubuntu-latest
-    needs: [check-files, check-is-hotfix]
-    if: needs.check-files.outputs.run_tests == 'true' && needs.check-is-hotfix.outputs.is_hotfix == 'false'
+    needs: [check-files]
+    if: |
+      (needs.check-files.outputs.run_tests == 'true' && ${{ !contains(github.event.pull_request.title, 'hotfix') }}) ||
```

Comment:
if you are using this `contains()` directly, you might as well delete `IS_HOTFIX` from L24